osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/mapping/DataObjectToEventListConverter.java

```diff
@@ -86,8 +109,19 @@ private DateTime extractDateTime(final List<DataObject> eventData) throws Protoc
         return dateTime;
     }
 
-    private Short extractCode(final List<DataObject> eventData) throws ProtocolAdapterException {
+    private Long extractEventDuration(EventLogCategoryDto eventLogCategory, List<DataObject> eventData)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "eventLogCategory". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)